content/cloud-docs/users-teams-organizations/teams.mdx

```diff
@@ -87,6 +87,8 @@ To add a teammate who doesn't yet belong to the organization, [invite them to jo
 
 To remove a user from a team, click the "🗑" (trash can) button by their entry in the member list.
 
+Customers in the Terraform Cloud Business tier may also manage team membership through their SSO provider. If [Managing Team Membership Through SSO](/cloud-docs/users-teams-organizations/single-sign-on#managing-team-membership-through-sso) is configured, users will be automatically added to teams corresponding to the Groups which they belong to in their SSO configuration.
```

Comment:
Also, I think we should move this so it comes right after the "to add" section. I think it makes more sense grouped with that. Let me know if you disagree!

```suggestion
If the organization is configured to [manage team membership through SSO](/cloud-docs/users-teams-organizations/single-sign-on#managing-team-membership-through-sso), users will be automatically added to teams corresponding to the groups they belong to in their SSO configuration.
```

Reply:
Agreed and updated, thanks as always for all of the suggestions @laurapacilio !